core/src/components/searchbar/test/searchbar.spec.ts

```diff
@@ -3,13 +3,37 @@ import { newSpecPage } from '@stencil/core/testing';
 import { Searchbar } from '../searchbar';
 
 describe('searchbar: rendering', () => {
-  it('should inherit attributes', async () => {
+  it('should inherit properties on load', async () => {
```

Comment:
I did not test updating the properties because that would be testing Stencil behavior. I am, however, testing updating attributes to verify that I configured the watchers correctly.